packages/video_player_avplay/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.3
+
+* Update plusplayer to latest.
```

Comment:
I don't know about the plusplayer package, but
Can you write a little more detail about the latest version?
version? latest commit information?

Reply:
I have added dedail in changelog.